docs/pages/examples.tsx

```diff
@@ -7,28 +7,50 @@ interface Props {}
 
 const Examples: NextPage<Props> = () => (
   <>
-    <div className="example">
+    <div className="prose dark:prose-invert">
+      <h1>Examples</h1>
+
       <h2>Task Manager App</h2>
-      <iframe
-        src="https://codesandbox.io/embed/dawn-cookies-yhsxo9?fontsize=14&hidenavigation=1&theme=dark"
-        style={{
-          width: '100%',
-          height: '800px',
-          border: '0',
-          borderRadius: '4px',
-          overflow: 'hidden',
-        }}
-        title="Task Manager App"
-        sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
-      ></iframe>
+      <img src="/images/task_manager_app.png" />
+      <p>A Kanban-style task manager.</p>
+      <ul>
+        <li>
+          <a href="https://codesandbox.io/embed/dawn-cookies-yhsxo9">
+            Running demo w/editable code
```

Comment:
Can we change the wording of this link to full words (ie. "with editable" instead of "w/editable" ) so it's more compatible with a screen reader 👍🏻

Reply:
Thanks!